docusaurus/video/docusaurus/docs/api/basics/get_started.mdx

```diff
@@ -567,8 +596,7 @@ call.queryRecordings("<session ID>");
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+recordings = call.query_recordings(sessionid="your_session_id")
```

Comment:
let's pick a better example, in most cases you dont know what the session_id is (missing an _ there btw)
I think a better example for this is to show first page and getting the next page

Reply:
We currently don't do pagination
<img width="1029" alt="Capture d’écran 2023-07-12 à 13 00 46" src="https://github.com/GetStream/protocol/assets/18029834/407261e6-bc84-444d-8fa0-acc3ad6cd622">